api/v1/schemas/google_oauth.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+from pydantic import BaseModel
+
+class UserData(BaseModel):
+    """
+    Schema Response representing the validated google login
+    """
+    id: str
+    first_name: str
+    last_name: str
+    username: str
+    email: str
```

Comment:
```suggestion
    email: EmailStr
```
needs to be imported from Pydantic

Reply:
The screenshots section is empty... @johnson-oragui

Reply:
> The screenshots section is empty... @johnson-oragui

It's difficult testing it with postman, postman uses a different Oauth flow from the Authlib module. But I will add the screenshots for testing with the browser, and keep trying with postman.

Thank you.